politeiawww/backend.go

```diff
@@ -1306,6 +1348,13 @@ func (b *backend) ProcessLogin(l www.Login) (*www.LoginReply, error) {
 		return nil, err
 	}
 
+	// Check if user is locked due to many login attempts
+	if user.UserLocked {
```

Comment:
I don't think we should check and report that the user is locked here, because if the user is locked, it tells the client that the email exists in the system, which we don't want to do. So we should only tell the user that his account is locked if he's entered the correct email+password combo. Otherwise we still report that it's an invalid email or password.